specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1153,6 +1161,47 @@
           }
         }
       }
+    },
+    "/{nextLink}": {
+      "post": {
+        "operationId": "PolicyStates_NextLink",
```

Comment:
The path probably need to be `{nextLink}` (without the `/`), as response have full URI.
`"@odata.context" : "https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/providers/Microsoft.PolicyInsights/policyStates/$metadata#latest/$entity"`

However the problem is that nextLink request should always be in GET, see https://github.com/Azure/azure-rest-api-specs/issues/15136